packages/network/src/index.ts

```diff
@@ -1,9 +1,17 @@
 import { TransactionVersion, ChainID, fetchPrivate } from '@stacks/common';
 
+export const HIRO_MAINNET_DEFAULT = 'https://stacks-node-api.mainnet.stacks.co';
+export const HIRO_TESTNET_DEFAULT = 'https://stacks-node-api.testnet.stacks.co';
+export const HIRO_MOCKNET_DEFAULT = 'http://localhost:3999';
+
+export interface NetworkUrl {
```

Comment:
```suggestion
export interface NetworkConfig {
```

I think this'd be a better name, as it could contain other properties.